pennylane/qchem/openfermion_obs.py

```diff
@@ -844,7 +844,6 @@ def molecular_hamiltonian(*args, **kwargs):
             For type dict, only int-keyed dict (for qubit-to-wire conversion) is accepted for
             partial mapping. If None, will use identity map.
         args (array[array[float]]): initial values of the differentiable parameters
-        load_data (bool): flag to load data from the basis-set-exchange library
```

Comment:
Can you no longer use `load_data` if you are using the `symbols` and `geometry` instead of `molecule` i.e. the old UI?

Reply:
load_data is an argument related to Molecule class, so passing the molecule object takes care of this argument.

Reply:
Users are still be able to use `load_data` with the old UI. With the new UI, it is absorbed in the molecule object.